packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -85,19 +86,40 @@ export const App = ({
   const [isSignedIn, setIsSignedIn] = useState<boolean>();
   const { loading, error } = useInitialSettingsState();
   const account = useAccount();
+  const integration = useIntegration();
   const { metricsEnabled } = account;
 
   // TODO Remove feature flag and experiment logic in FXA-7419
   const showRecoveryKeyV2 = !!(
     config.showRecoveryKeyV2 && isInRecoveryKeyExperiment === 'true'
   );
 
+  useMemo(() => {
```

Comment:
Curious why `useMemo` over `useEffect`? When I've seen useMemo, it typically uses memoize value somewhere.

Reply:
`useEffect` executes at the _end_ of a render, which means _after_ all the children have been rendered, but some of those children might need to use Glean.  This ensures that Glean is initialized before the children components use it.

`useMemo` seemed the most appropriate of the React provided hooks.